ci/launch_cluster.sh

```diff
@@ -42,21 +42,30 @@ installer_url: $INSTALLER
 provider: onprem
 platform: aws
 aws_region: us-west-2
-os_name: cent-os-7-dcos-prereqs
 key_helper: true
 instance_type: m4.large
-num_public_agents: 1
-num_private_agents: 3
 num_masters: 3
+os_name: cent-os-7-dcos-prereqs
+ssh_user: centos
 dcos_config:
     cluster_name: $DEPLOYMENT_NAME
     resolvers:
-        - 8.8.4.4
         - 8.8.8.8
-    dns_search: mesos
+        - 8.8.4.4
+    dns_search: us-west-2.compute.internal
```

Comment:
Why is this required?

Reply:
I don't really remember. I only remember that I had to go through some try and error until I got something working... But let's try what we are doing right now on `master` and if it fails we try my defaults 👍